src/components/screens/question-form.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  Box,
+  Button,
+  Card,
+  CardActionArea,
+  CardActions,
+  CardContent,
+  Checkbox,
+  TextField,
+  Typography
+} from "@mui/material";
+import type React from "react";
+import { useState } from "react";
+
+/**
+ * NewQuestionCard component
+ * Interface for the NewQuestionCard component
+ */
+
+interface NewQuestionProps {
+  handleAddQuestionSubmit: (
+    questionText: string,
+    options: { label: string; value: boolean }[]
+  ) => void;
+}
+
+const NewQuestionCard: React.FC<NewQuestionProps> = ({ handleAddQuestionSubmit }) => {
+  /**
+   * State for the question text and options
+   */
+  const [questionText, setQuestionText] = useState("");
+  const [options, setOptions] = useState([
+    { label: "", value: false },
+    { label: "", value: false },
+    { label: "", value: false },
+    { label: "", value: false }
```

Comment:
I suspect there is a cleaner way of doing this. Perhaps also initializing only one option and allowing the user to add options through the UI.

Reply:
There is possibility to add more. At this point, there are 4 as default, but we can make it cleaner by start with one.